packages/monorepo-builder/packages/Merge/Command/MergeCommand.php

```diff
@@ -55,4 +56,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         return self::SUCCESS;
     }
+
+    private function getMainComposerJson(string $mainComposerJsonFilePath): ComposerJson
```

Comment:
We use convention "root" for the top most `composer.json` file that we merge to.

Reply:
I've used the convention from the `execute` method. Should I change "main" to "root" there too?

Reply:
I see. Yes, that would be helpful 👍